### DISCUSSION TITLE
Redirecting to a page when select a date

### DISCUSSION THREAD

**Discussion by anindyoy** [saade/filament-fullcalendar#204] [Q&A]
How can I open a create record of the resource page if I select the date, with like mountUsing method or sending date_start as parameter?

**Comment by useahawk** [comment]
Did you find a solution fpr this?

**Reply by anindyoy** [reply]
sadly, nope

**Reply by abdulmannans-joistic** [reply]
any solution on this?

**Reply by anindyoy** [reply]
not yet
